clients/cloud/java-springboot/README.adoc

```diff
@@ -0,0 +1,172 @@
+= Spring Boot with Java Example Confluent Cloud Client
+
+NOTE: Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) using the Spring Boot using Java API.
+
+== Prerequisites
+
+* Java 1.8 or higher to run the demo application
```

Comment:
`gradle` is a prereq

Reply:
we use Gradle wrapper. Gradle isn't prerequisite in this case, because wrapper will download required version of gradle